app/main/helpers/briefs.py

```diff
@@ -29,6 +29,26 @@ def domain(email):
     current_user_domain = domain(current_user.email_address) \
         if domain(current_user.email_address) not in current_app.config.get('GENERIC_EMAIL_DOMAINS') \
         else None
+    if brief.get('lot', '') == 'atm':
+        if hasattr(current_user, 'role') and current_user.role == 'supplier':
+            brief_category = brief.get('sellerCategory', '')
+            supplier = data_api_client.get_supplier(current_user.supplier_code)['supplier']
+            if (
+                (
+                    supplier and brief.get('openTo', '') == 'all' and len(supplier['domains']['assessed']) > 0
+                ) or
+                (
+                    supplier and
+                    brief.get('openTo', '') == 'category' and
+                    brief.get('areaOfExpertise', '') and
+                    brief.get('areaOfExpertise', '') in supplier['domains']['assessed']
+                )
+            ):
```

Comment:
I think there's some repetition we can remove which also helps with the formatting.  Does this look a bit better?
```python
open_to = brief.get('openTo', '')
area_of_expertise = brief.get('areaOfExpertise', '')
if (supplier and
   (open_to == 'all' and len(supplier['domains']['assessed']) > 0) or
   (open_to == 'category' and area_of_expertise and area_of_expertise in supplier['domains']['assessed'])):
```